test/models/project_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class ProjectTest < ActiveSupport::TestCase
+  test "valid with user" do
+    user = User.create(name: "Ali", mobile: "0900", email: "a2@g.com", password_digest: "123", address: "ajhsncjfacnaje")
```

Comment:
Are we writing test for Project ? I am not seeing anything being validated on project model in this test case.

Reply:
done